File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        String ret = "";
        
        for(char c : rsp.toCharArray()){
            if(c == '2') ret += '0';
            else if(c == '0') ret += '5';
            else if(c == '5') ret += '2';
        }
        
        return ret;
    }
}
//

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.cpp
#include<bits/stdc++.h>

using namespace std;
map<char, char> mp;
string solution(string rsp) {
    mp['2'] = '0';
    mp['0'] = '5';
    mp['5'] = '2';
    string ret = "";
    for(char c : rsp) ret += mp[c];
    return ret;
}